### ISSUE TITLE
Could not load file or assembly Microsoft.VisualStudio.Threading, Version=16.4.0.0

### ISSUE THREAD

**Issue by fr4gles** [pmiossec/Gitextensions.AzureDevOpsCommitMessage#1]
Hello there,
I have encoutered error while using your plugin.
In fact I cannot use it due to this error.

## Current behaviour
I've tried to open newly installed plugin.

## Expected behaviour
Plugins options opens without exception

## Steps to reproduce
open git extesions after install -> Pluins -> click on plugin AzureDevOpsCommitMessage

## Error Details
```
System.IO.FileNotFoundException: Could not load file or assembly 'Microsoft.VisualStudio.Threading, Version=16.4.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' or one of its dependencies. The system cannot find the file specified.
File name: 'Microsoft.VisualStudio.Threading, Version=16.4.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'
   at GitExtensions.AzureDevOpsCommitMessage.Plugin.Execute(GitUIEventArgs args)
   at GitUI.CommandsDialogs.FormBrowse.<>c__DisplayClass62_0.<RegisterPlugins>b__2(Object <p0>, EventArgs <p1>)
   at System.Windows.Forms.ToolStripItem.RaiseEvent(Object key, EventArgs e)
   at System.Windows.Forms.ToolStripMenuItem.OnClick(EventArgs e)
   at System.Windows.Forms.ToolStripItem.HandleClick(EventArgs e)
   at System.Windows.Forms.ToolStripItem.HandleMouseUp(MouseEventArgs e)
   at System.Windows.Forms.ToolStripItem.FireEventInteractive(EventArgs e, ToolStripItemEventType met)
   at System.Windows.Forms.ToolStripItem.FireEvent(EventArgs e, ToolStripItemEventType met)
   at System.Windows.Forms.ToolStrip.OnMouseUp(MouseEventArgs mea)
   at System.Windows.Forms.ToolStripDropDown.OnMouseUp(MouseEventArgs mea)
   at System.Windows.Forms.Control.WmMouseUp(Message& m, MouseButtons button, Int32 clicks)
   at System.Windows.Forms.Control.WndProc(Message& m)
   at System.Windows.Forms.ScrollableControl.WndProc(Message& m)
   at System.Windows.Forms.ToolStrip.WndProc(Message& m)
   at System.Windows.Forms.ToolStripDropDown.WndProc(Message& m)
   at System.Windows.Forms.Control.ControlNativeWindow.OnMessage(Message& m)
   at System.Windows.Forms.Control.ControlNativeWindow.WndProc(Message& m)
   at System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)

WRN: Assembly binding logging is turned OFF.
To enable assembly bind failure logging, set the registry value [HKLM\Software\Microsoft\Fusion!EnableLog] (DWORD) to 1.
Note: There is some performance penalty associated with assembly bind failure logging.
To turn this feature off, remove the registry value [HKLM\Software\Microsoft\Fusion!EnableLog].

```

## Environment
- Git Extensions 3.4.0.9381
- Build 096b4d56709e15a67b42a471bbe81ce3c95ae7f5
- Git 2.26.2.windows.1
- Microsoft Windows NT 6.2.9200.0
- .NET Framework 4.8.4084.0
- DPI 96dpi (no scaling)

**Comment by pmiossec** [comment]
It's a known issue with the last (not yet released) version of GitExtensions (Issue tracking: https://github.com/gitextensions/gitextensions/issues/8003). 

I have to do a new release of the plugin.

A workaround is to add these lines to the `GitExtensions.exe.config` file (next to other `assemblyBinding` sections):

```xml
    <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
      <dependentAssembly>
        <assemblyIdentity name="Microsoft.VisualStudio.Threading" publicKeyToken="b03f5f7f11d50a3a" culture="neutral" />
        <bindingRedirect oldVersion="0.0.0.0-16.5.0.0" newVersion="16.5.0.0" />
      </dependentAssembly>
    </assemblyBinding>
```

And it should work.

**Comment by pmiossec** [comment]
Should be fixed with v1.1.0
